app/web/features/messages/messagelist/MessageList.tsx

```diff
@@ -35,34 +34,32 @@ export default function MessageList({
   className,
 }: MessageListProps) {
   const { t } = useTranslation(MESSAGES);
-  const classes = useStyles();
 
   return (
-    <div
-      className={classNames(classes.list, className)}
-      data-testid="message-list"
-    >
+    <List className={classNames(className)} data-testid="message-list">
       {messages.length ? (
         messages.map((message) =>
           isControlMessage(message) ? (
-            <ControlMessageView
-              key={message.messageId}
-              onVisible={() => markLastSeen(message.messageId)}
-              message={message}
-              className={classes.message}
-            />
+            <MessageWrapper key={message.messageId}>
+              <ControlMessageView
+                key={message.messageId}
+                onVisible={() => markLastSeen(message.messageId)}
+                message={message}
+              />
+            </MessageWrapper>
           ) : (
-            <MessageView
-              key={message.messageId}
-              onVisible={() => markLastSeen(message.messageId)}
-              message={message}
-              className={classes.message}
-            />
+            <MessageWrapper key={message.messageId}>
+              <MessageView
+                key={message.messageId}
+                onVisible={() => markLastSeen(message.messageId)}
+                message={message}
+              />
+            </MessageWrapper>
```

Comment:
```suggestion
              <StyledMessageView
                key={message.messageId}
                onVisible={() => markLastSeen(message.messageId)}
                message={message}
              />
```

Make a separate styled for this one. I know it's duplicating but alas.